cmd/loadtest/loadtest.go

```diff
@@ -52,6 +52,11 @@ var tests []TestItem = []TestItem{
 		ShortDesc: "Test leaving and joining a team while under load",
 		Test:      &loadtest.TestLeaveJoinTeam,
 	},
+	{
+		Name:      "autocomplete",
```

Comment:
Would it make sense to eventually converge on a definition of "test all" that tests all these types of actions with some reasonable distribution between the entities?

Reply:
That's the goal of "test all"

Reply:
Ah, of course, I hadn't fully dived into the definition of all these structs and how they are used. 👍